### ISSUE TITLE
LOCO computes all SNPs if no markers on chr X

### ISSUE THREAD

**Issue by pjotrp** [genetics-statistics/GEMMA#145]
In one set we have no markers on a chromose and it fails to do a proper LOCO (i.e., skip results).

See https://github.com/genetics-statistics/GEMMA/blob/master/src/lmm.cpp#L203

**Comment by pjotrp** [comment]
This happens when chromosome names are not recognised.

**Comment by danfulop** [comment]
@pjotrp  Is the workaround for this to include the X chromosome?

**Comment by pjotrp** [comment]
Yes, a few values will do, and use expected names for chromosomes. The problem is really that if a name is not found it will include all values in the output (like running non-LOCO). No harm done. I have not gotten round to fixing yet (LOCO is a first rate citizen in the upcoming edition).

**Comment by danfulop** [comment]
I have mouse data...  So, are 1-19 + X good/expected chromosome names?

**Comment by pjotrp** [comment]
We run mouse data, so it should work.

**Comment by pjotrp** [comment]
Will be automatic in gemma2/lib
